src/components/state-details.js

```diff
@@ -94,28 +94,251 @@ const getPlacesData = (data) => {
   return allPlacesData
 }
 
+const currentYear = new Date().getFullYear()
+
+// We want to get to 0 by 2050 and we use our current emissions as a start,
+// so the % to cut by is 100 divided by the number of years we have
+const cutPerYearPrcnt = (100 / (2050 - currentYear)).toFixed(1)
```

Comment:
This will automatically tweak the cut % as the years progress, but keep in mind this means in 2040 for example this would say 25% no matter what. This is, technically, accurate but I think a bit misleading.